config.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  cluster_name:
+    description: "Optional - Name of the MySQL InnoDB cluster"
+    type: "string"
```

Comment:
Why generate a random cluster name on the charm instead of just having a default name here?

Reply:
One can have multiple database clusters on the same k8s cluster - having no default can avoid collisions.